tests/test/transport/test_point_to_point.cpp

```diff
@@ -143,6 +144,85 @@ TEST_CASE_METHOD(PointToPointClientServerFixture,
     conf.reset();
 }
 
+TEST_CASE_METHOD(PointToPointClientServerFixture,
+                 "Test point-to-point in-order message delivery",
+                 "[transport][ptp]")
+{
+    int appId = 123;
+    int groupId = 345;
+    int idxA = 0;
+    int idxB = 1;
+
+    // Ensure this host is set to localhost
+    faabric::util::SystemConfig& conf = faabric::util::getSystemConfig();
+    conf.endpointHost = LOCALHOST;
+
+    // Register both indexes on this host
+    faabric::util::SchedulingDecision decision(appId, groupId);
+
+    faabric::Message msgA = faabric::util::messageFactory("foo", "bar");
+    msgA.set_appid(appId);
+    msgA.set_groupid(groupId);
+    msgA.set_groupidx(idxA);
+
+    faabric::Message msgB = faabric::util::messageFactory("foo", "bar");
+    msgB.set_appid(appId);
+    msgB.set_groupid(groupId);
+    msgB.set_groupidx(idxB);
+
+    decision.addMessage(LOCALHOST, msgA);
+    decision.addMessage(LOCALHOST, msgB);
+
+    // Set up the mappings and configure in-order delivery
+    broker.setAndSendMappingsFromSchedulingDecision(decision);
+    broker.setMustOrderMessages(true);
```

Comment:
I would expect this test to have two sections, one with ordering on, one with ordering off, then check that the sender is setting the sequence numbers right (this can still be checked without the server being used right?).

Reply:
Yes, the sequence numbers are implicitly checked by running the tests with the ordering on. Do you mean explicitly checking for the sequence number in the test? 

Irrespective of your answer, I add the requested section in the test.